### DISCUSSION TITLE
cargo: how to install with support for dataframes, for current version

### DISCUSSION THREAD

**Discussion by aborruso** [nushell/nushell#7336] [Q&A] [answered]
Hi,
I have installed nu, using `cargo install nu --features=dataframe`, but I get `0.61.0`.

How to force cargo to install the current version?

Thank you

**Comment by aborruso** [comment, marked as answer]
Hi,
I have solved in this way

```
git clone https://github.com/nushell/nushell.git
cd nushell 
cargo install --path=. --all-features
```

/via discord https://discord.com/channels/601130461678272522/601130461678272524/1048560036839964795

**Reply by fdncred** [reply]
I use the install-all.sh script in the root of the repo which builds all plugins and does a `cargo install --path . --features=dataframe`
